.github/workflows/harvest.yml

```diff
@@ -0,0 +1,27 @@
+name: Harvest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - 0 4 * * *
+
+jobs:
+  harvest:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: pjquirk/harvest-action@main
```

Comment:
This workflow should be moved somewhere other than my personal account, to avoid bus-factor issues.  Since this is so tied into Linguist, you might prefer having it in this repo under a top-level `/actions` folder.